upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.6.yaml.template

```diff
@@ -55,7 +61,6 @@ data:
   Corefile: |
     .:53 {
         errors
-        log
```

Comment:
❤️  Thank you!! Logging every DNS request is so verbose, I've been removing this manually on every cluster.